v2/app/src/main/java/com/google/firebase/example/makeitso/data/repository/TodoItemRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.google.firebase.example.makeitso.data.repository
+
+import com.google.firebase.example.makeitso.data.datasource.TodoItemRemoteDataSource
+import com.google.firebase.example.makeitso.data.model.TodoItem
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class TodoItemRepository @Inject constructor(
+    private val todoItemRemoteDataSource: TodoItemRemoteDataSource
+) {
+    val todoItems: Flow<List<TodoItem>> = todoItemRemoteDataSource.getTodoItems("")
```

Comment:
Is the list id supposed to be an empty string here?

Reply:
No, I'll add this after adding the UI - this ID will be passed when navigating to this screen, and the "onClick" function is not yet implemented.